net-misc/r8152/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>jay@jvf.cc</email>
+		<name>Jay Faulkner</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>Official Realtek r8152 linux driver. The following cards are currently supported:
+RTL8156 RTL8156B(S)(G) RTL8153 RTL8153B RTL8154 RTL8154B RTL8152B</longdescription>
+	<use>
+		<flag name="center-tap-short">Enable support for center tap short</flag>
```

Comment:
What does this actually mean?

Reply:
This is verbatim from the upstream documentation. -> https://github.com/wget/realtek-r8152-linux/blob/master/ReadMe.txt#L31 I have no idea. I tried to figure it out, I still have no idea. The ebuild defaults to the behavior if you do no configuration at all.